src/Randomizer.SMZ3/Regions/Zelda/SkullWoods.cs

```diff
@@ -56,7 +56,7 @@ public SkullWoods(World world, Config config) : base(world, config)
             BigKeyChest = new Location(this, 256 + 150, 0x1E99E, LocationType.Regular,
                 name: "Big Key Chest",
                 vanillaItem: ItemType.BigKeySW,
-                memoryAddress: 0x39,
+                memoryAddress: 0x57,
```

Comment:
How do you figure out these addresses anyway? Is there a memory debugger or do you have a document somewhere that, well, documents these?